app/controllers/lessons_controller.rb

```diff
@@ -47,42 +47,46 @@ def new
   end
 
   def create
-    lesson = Lesson.new
-    lesson.hash_tag = params[:lesson][:hash_tag]
-    lesson.created_at = Time.now
-    lesson.category = params[:lesson][:category]
-    lesson.description = params[:lesson][:description]
-    lesson.approved = true
-    lesson.save
-    cookies["new_lesson_id"] = lesson.id
+    if user_signed_in?
+      lesson = Lesson.new
+      lesson.hash_tag = params[:lesson][:hash_tag]
+      lesson.created_at = Time.now
+      lesson.category = params[:lesson][:category]
+      lesson.description = params[:lesson][:description]
+      lesson.approved = true
+      lesson.save
+      cookies["new_lesson_id"] = lesson.id
 
     # create a new contribution for creator
-    contribution = Contribution.new
-    contribution.lesson_id = lesson.id
-    contribution.user_id = session["user_id"]
-    contribution.creator = true
-    contribution.save
+      contribution = Contribution.new
+      contribution.lesson_id = lesson.id
+      contribution.user_id = session["user_id"]
+      contribution.creator = true
+      contribution.save
+    end
```

Comment:
This indentation is off, it should be corrected by ensuring this end lines up with the next line, and that everything in the if is indented inside that. It might just be the next line has too much indent.